apps/sched/interface.html

```diff
@@ -150,12 +200,53 @@
   document.getElementById('upload').disabled = false;
 }
 
+function msToHMS(ms) {
+  let secs = Math.floor(ms / 1000) % 60;
+  let mins = Math.floor(ms / 1000 / 60) % 60;
+  let hours = Math.floor(ms / 1000 / 60 / 60);
+  if (secs < 10) secs = "0" + secs;
+  if (mins < 10) mins = "0" + mins;
+  if (hours < 10) hours = "0" + hours;
+  return [hours, mins, secs];
+}
+
+function hmsToMs(hms) {
+  let [hours, mins, secs] = hms.split(":");
+  hours = Number(hours);
+  mins = Number(mins);
+  secs = Number(secs);
+  return ((hours * 60 + mins) * 60 + secs) * 1000;
+}
+
 function addAlarm() {
   const alarm = getAlarmDefaults();
   renderAlarm(alarm);
   alarms.push(alarm);
 }
 
+function addDailyAlarm() {
+  const alarm = getAlarmDefaults();
+  delete alarm.date;
+  renderAlarm(alarm);
+  alarms.push(alarm);
+}
+
+function addTimer() {
+  const alarmDefaults = getAlarmDefaults();
+  const timer = {
+    timer: hmsToMs("00:00:30"),
+    on: false,
```

Comment:
Is there a reason why it defaults to disabled?

Reply:
No strong opinion - I was thinking a user might want to populate with a few default timers but thinking about it, defaulting to on makes more sense.